.gitignore

```diff
@@ -1,24 +1,4 @@
-# Docker project generated files to ignore
-#  if you want to ignore files created by your editor/tools,
-#  please consider a global .gitignore https://help.github.com/articles/ignoring-files
-*.exe
-*.exe~
-*.gz
-*.orig
-test.main
-.*.swp
-.DS_Store
-# a .bashrc may be added to customize the build environment
-.bashrc
-.editorconfig
-.gopath/
-.go-pkg-cache/
+# IDE files and folders
+.project
+.settings/
```

Comment:
just ignore all dot files

Reply:
is there a good .gitignore you would recommend? i got some of this from either docker/docker or docker/libnetwork

Reply:
In moby/moby, we added this line; https://github.com/moby/moby/blob/675b414f56063826457cbc8f0c165af67198bfa8/.gitignore#L2-L3

mostly because we didn't want to update the file for each-and-every IDE out there.
We should probably have slightly more instructions (e.g. explain how to add it too `./.git/info/exclude` to exclude things for a single repository, without having to update the `.gitignore` itself)